File: IoTEx.WaternetIoT/IoTEx.Waternet.Portal/src/app/models/DeviceTelemetryAggregateDTO.ts
import { BaseDTO } from "./baseDTO";



export class DeviceTelemetryAggregateDTO extends BaseDTO {

  name?: string;
  windowIdx?: number;
  windowDT?: Date;
  min?: number;
  max?: number;
  count?: number;
  stdDev?: number;
  avg?: number;
  pct90?: number;
}

File: IoTEx.WaternetIoT/IoTEx.WaternetIoT.Model/DTOs/DeviceTelemetryAggregateDTO.cs
using Newtonsoft.Json;
using System;
using System.Collections.Generic;
using System.Text;

namespace IoTEx.WaternetIoT.Model.DTOs
{
    public class DeviceTelemetryAggregateDTO
    {
        public DateTime WindowDT { get; set; }
        public string Name { get; set; }
        public Int64 Count { get; set; }
        public double Min { get; set; }
        public double Max { get; set; }
        public double Sum { get; set; }
        public double Avg { get; set; }
        public double StdDev { get; set; }
        public double Pct90 { get; set; }
        public long WindowIdx { get; set; }

    }
}
